src/globus_sdk/experimental/login_flow_manager/command_line_login_flow_manager.py

```diff
@@ -0,0 +1,92 @@
+from globus_sdk import AuthLoginClient, OAuthTokenResponse
+from globus_sdk.experimental.auth_requirements_error import (
+    GlobusAuthorizationParameters,
+)
+
+from .login_flow_manager import LoginFlowManager
+
+
+class CommandLineLoginFlowManager(LoginFlowManager):
+    """
+    A ``CommandLineLoginFlowManager`` is a ``LoginFlowManager`` that uses the command
+    line for interacting with the user during its interactive login flows.
+
+    Example usage:
+
+    >>> login_client = globus_sdk.NativeAppAuthClient(...)
+    >>> login_flow_manager = CommandLineLoginFlowManager(login_client)
+    >>> scopes = [globus_sdk.scopes.TransferScopes.all]
+    >>> auth_params = GlobusAuthorizationParameters(required_scopes=scopes)
+    >>> tokens = login_flow_manager.run_login_flow(auth_params)
+
+    """
+
+    def __init__(
+        self,
+        login_client: AuthLoginClient,
+        *,
+        login_prompt: str = "Please authenticate with Globus here:",
+        code_prompt: str = "Enter the resulting Authorization Code here:",
+    ):
+        """
+        :param login_client: The ``AuthLoginClient`` that will be making the Globus
+            Auth API calls needed for the authentication flow. Note that this
+            must either be a NativeAppAuthClient or a templated
+            ConfidentialAppAuthClient, standard ConfidentialAppAuthClients cannot
+            use the web auth-code flow.
+        :param login_prompt: The string that will be output to the command line
+            prompting the user to authenticate.
+        :param code_prompt: The string that will be output to the command line
+            prompting the user to enter their authorization code.
+        """
+        self.login_prompt = login_prompt
+        self.code_prompt = code_prompt
+        super().__init__(login_client)
+
+    def run_login_flow(
+        self,
+        auth_parameters: GlobusAuthorizationParameters,
+        *,
+        refresh_tokens: bool = False,
```

Comment:
Would this make more sense as a class init param?

(Thinking out loud more than making a suggestion)

I think that builds more in the direction of "I'm operating a script/tool which deals with refresh tokens so I configure my login manager to use refresh tokens".

Reply:
Hmm, yeah, I think that lines up better with `GlobusApp` having it configured at the app level rather than at individual calls to `run_login_flow` I'll move it!